### ISSUE TITLE
The <input /> rendered by MultipleFileUploadEditor is missing multiple attribute

### ISSUE THREAD

**Issue by wezmag** [serenity-is/Serenity#6646]
### Before submitting the bug report, please read and check the following items

- [X] This isn't a feature request or a discussion topic (Please use discussions for that)
- [X] I have searched open and closed issues for duplicates
- [X] I have searched the documentation on the Serenity Platform website https://serenity.is/docs/
- [X] I have read the frequently asked questions https://serenity.is/docs/startsharp/faq
- [X] I have searched my problem in the discussions and the old issues https://github.com/serenity-is/Serenity/discussions

### What happened?

After migrating my project to the latest version, I was not able to select multiple files at once, when using `MultipleFileUploadEditor`.
After dig into the issue, I found out that the rendered `<input />` tag is missing `multiple` attribute, which is different from the previous version.

### What did you expect to happen?

When using `MultipleFileUploadEditor`, instead of selecting one file at a time, I should be able to select multiple files at once.

### How to reproduce?

Use `MultipleFileUploadEditor` in a form.

### What Serenity Nuget Versions are you seeing the problem on? (separated by comma)

6.4.2

### Additional information

The `addUploadInput` function in the `UploadHelper` does check the options for `allowMultiple`.
```
https://github.com/serenity-is/Serenity/blob/master/src/Serenity.Scripts/corelib/src/ui/helpers/uploadhelper.ts#L23
```
However, the `getUploadInputOptions` function in the `MultipleFileUploadEditor` does not set `allowMultiple` to `true`.
```
https://github.com/serenity-is/Serenity/blob/master/src/Serenity.Scripts/corelib/src/ui/editors/uploadeditors.ts#L316
```

**Comment by VictorTomaili** [comment]
Use discussions, please.

**Comment by wezmag** [comment]
> Use discussions, please. 

I thought it's a bug, isn't it?

**Comment by VictorTomaili** [comment]
Thank you for bringing this issue to my attention. After reviewing the case again, I can confirm that it is not currently possible to select multiple files at the same time in the multiple file upload feature. However, users can upload multiple files through drag-and-drop or by selecting them one at a time. While this may not be a bug, it is a missing feature that we should address. I will investigate the reason for this limitation and keep this issue open for further examination.

**Comment by ibrahimGit123** [comment]
Hi Victor - any update on this? We are running into the same issue. Our users are used to selecting many files at the same time.

**Comment by VictorTomaili** [comment]
Not yet

**Comment by VictorTomaili** [comment]
Will be published with next serenity scripts version.

**Comment by wezmag** [comment]
Issue fix in v6.5.2.0
Thanks team!
